muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        out_base=\$(grep -A1 '^name' $mu_sim | grep -v 'name' | sed -e 's/^[ \t]*//') &&
+        out_base=\${out_base:-"muspinsim"} &&
+        log_file=\$(echo $mu_sim | cut -d '.' -f1).log &&
+        ln -s $mu_exp_in $mu_exp_in.name &&
+        if grep '^fitting_data' $mu_sim; then sed -iE "/^fitting_data/{n;s/([^)]*)/(\"$mu_exp_in.name\")/g}" $mu_sim; fi &&
+        muspinsim $mu_sim &&
+        mv \$out_base.dat mu_out.dat &&
+        mv \$log_file log_out.log
+    ]]></command>
+    <inputs>
+        <param type="data" name="mu_sim" label="input file containing simulation parameters"/>
+        <param type="data" name="mu_exp_in" optional="true" label="Experiment data to fit (.dat) (Optional)" />
+    </inputs>
+    <outputs>
+        <data label=".dat output file for $mu_sim.name" name="mu_out" format="txt" from_work_dir="mu_out.dat" />
```

Comment:
These output names should ideally communicate what the file contains - the fact it's a .dat doesn't tell a new user very much. I've added some suggestions but feel free to tweak them if you think something else is suitable.

```suggestion
        <data label="muspinsim results for $mu_sim.name" name="mu_out" format="txt" from_work_dir="mu_out.dat" />
```

Reply:
changes committed [eab0f8c](https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/pull/55/commits/eab0f8c5cb3ef93431c1668377477090c60a62b3)